Makefile

```diff
@@ -81,6 +81,12 @@ lint:
 vet:
 	VET_INPUT="$(shell go list ./... | grep -v /vendor/)"; GO111MODULE=on go vet $$VET_INPUT
 
+# TODO: improve the command to also check scripts outside the scripts and template-processors dirs
+.PHONY: check-shfmt
+check-shfmt:
+	test -z "$(shell shfmt -i 4 -l ./scripts)"
+	test -z "$(shell shfmt -i 4 -l ./template-processors)"
```

Comment:
Would it still work if we [used -d flag][flag] instead of wrapping in `test -z`?

      -d        error with a diff when the formatting differs

[flag]: https://github.com/mvdan/sh/blob/30598145b5d9322bf63da248563e34bda5e3df03/cmd/shfmt/main.go#L78

Reply:
Yep ... good idea. Fixed.